### PR TITLE
Fixes #9348: Add the shared file constraint to copy from shared folder techniques

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9348

### CODE REVIEW

File: techniques/fileDistribution/copyGitFile/2.1/metadata.xml [hunk 1]
```diff
@@ -47,7 +47,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       <INPUT>
         <NAME>COPYFILE_NAME</NAME>
         <DESCRIPTION>Path of the file to be copied</DESCRIPTION>
-      <LONGDESCRIPTION>This is the relative path of the file/folder to be copied, on the Rudder policy server</LONGDESCRIPTION>
+        <LONGDESCRIPTION>This is the relative path of the file/folder to be copied, on the Rudder policy server</LONGDESCRIPTION>
+        <CONSTRAINT>
+          <SHAREDFILE></SHAREDFILE>
```
Comment (VinceMacBuche) [inline techniques/fileDistribution/copyGitFile/2.1/metadata.xml]: Same here: Should be <TYPE>sharedfile</TYPE>

File: techniques/fileDistribution/copyGitFile/2.0/metadata.xml [hunk 2]
```diff
@@ -47,7 +47,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       <INPUT>
         <NAME>COPYFILE_NAME</NAME>
         <DESCRIPTION>Path of the file to be copied</DESCRIPTION>
-      <LONGDESCRIPTION>This is the relative path of the file/folder to be copied, on the Rudder policy server</LONGDESCRIPTION>
+        <LONGDESCRIPTION>This is the relative path of the file/folder to be copied, on the Rudder policy server</LONGDESCRIPTION>
+        <CONSTRAINT>
+          <SHAREDFILE></SHAREDFILE>
```
Comment (VinceMacBuche) [inline techniques/fileDistribution/copyGitFile/2.0/metadata.xml]: Should be <TYPE>sharedfile</TYPE>

Comment (ncharles) [inline techniques/fileDistribution/copyGitFile/2.0/metadata.xml]: should be <CONSTRAINT><TYPE>sharedfile</TYPE></CONSTRAINT>

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**